src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.scss

```diff
@@ -4,14 +4,70 @@
   width: 100%;
   display: flex;
   padding: 20px;
+  &.collapsed {
+    display: none;
+  }
 }
 
-.checkboxGridTitle {
+.checkboxGridHeader {
   color: $grey;
-  margin-bottom: 10px;
+  padding-top: 10px;
   padding-left: 20px;
+  position: relative;
+  height: 45px;
+}
+
+.checkboxGridHeader + .checkboxGridContainer {
+  padding-top: 5px;
+}
+
+.label {
+  color: $grey;
+  float: left;
```

Comment:
Float? Oh dear, do we really want floats?

Reply:
I've changed it to work with `display`. Just wondering why float is not good to use?

Reply:
It's not that float is not good to use; it's just that it was historically used (abused) to achieve all sorts of tasks when the browser didn't have any better alternatives. Nowadays, with the much more powerful CSS rules, we can reserve it for what it was originally intended — text-wrap effects.